src/proposal.mligo

```diff
@@ -221,8 +224,8 @@ let unfreeze_proposer_and_voter_token
 
 
 [@inline]
-let is_time_reached (proposal, sec : proposal * seconds): bool =
-  Tezos.now > proposal.start_date + int(sec)
+let is_level_reached (proposal, target : proposal * levels): bool =
+  Tezos.level > proposal.start_level.levels + target.levels
```

Comment:
I wonder if we should consider using `>=` now :thinking:

Reply:
If a proposal is raised in 100th level, and say proposal exipiry is 5 levels, then I think the expectation is that it should be unexpired for 5 levels. So I think it should expire on 106th level. So shouldn't `>` is the right one itself?

Reply:
:thinking: the way we define it in the config:
> Determine the minimum number of levels after the proposal is proposed to be considered as expired.

I think you are right, as this reads to me as "_after_ 5 level from the one it was created in, this is expired" :ok_hand:

Reply:
@pasqu4le,  After giving it more thought, it appears that you maybe correct as well. The question really come down to whether we want to count blocks like we count seconds. If it is like seconds, then we should use `>`. If it is counted as discreete entities (which I think we should), then it should be `>=`.

Either way, I think we need to clarify this in the docs to guard against the possibility of off-by-one errors.